info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T04:32:49.008Z"}
info: Connected to the database. {"timestamp":"2022-02-09T04:32:49.075Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: Books.users index: email_1 dup key: { email: "js4770526@gmail.com" } 
      url - /api/v1/users/user 
      method - POST 
      IP - ::1
     {"timestamp":"2022-02-09T04:37:18.523Z"}
error: 
    status - 500 
    message - RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: 11000
    at new NodeError (node:internal/errors:371:5)
    at ServerResponse.writeHead (node:_http_server:275:11)
    at ServerResponse.writeHead (/home/saeb/Book_Store/Book_Store/node_modules/on-headers/index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:266:8)
    at write_ (node:_http_outgoing:763:9)
    at ServerResponse.end (node:_http_outgoing:852:5)
    at ServerResponse.send (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/response.js:221:10)
    at ServerResponse.json (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/response.js:267:15)
    at appErrorHandler (/home/saeb/Book_Store/Book_Store/build/middlewares/error.middleware.js:43:26)
    at Layer.handle_error (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:321:13)
    at /home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:341:12)
    at Immediate.next (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:275:10)
    at Immediate._onImmediate (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:641:15)
    at processImmediate (node:internal/timers:466:21) 
    url - /api/v1/users/user 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-09T04:37:18.527Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T04:41:16.221Z"}
info: Connected to the database. {"timestamp":"2022-02-09T04:41:16.274Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: Books.users index: email_1 dup key: { email: "js4770526@gmail.com" } 
      url - /api/v1/users/admin 
      method - POST 
      IP - ::1
     {"timestamp":"2022-02-09T04:42:38.004Z"}
error: 
    status - 500 
    message - RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: 11000
    at new NodeError (node:internal/errors:371:5)
    at ServerResponse.writeHead (node:_http_server:275:11)
    at ServerResponse.writeHead (/home/saeb/Book_Store/Book_Store/node_modules/on-headers/index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:266:8)
    at write_ (node:_http_outgoing:763:9)
    at ServerResponse.end (node:_http_outgoing:852:5)
    at ServerResponse.send (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/response.js:221:10)
    at ServerResponse.json (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/response.js:267:15)
    at appErrorHandler (/home/saeb/Book_Store/Book_Store/build/middlewares/error.middleware.js:43:26)
    at Layer.handle_error (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:321:13)
    at /home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:341:12)
    at Immediate.next (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:275:10)
    at Immediate._onImmediate (/home/saeb/Book_Store/Book_Store/node_modules/express/lib/router/index.js:641:15)
    at processImmediate (node:internal/timers:466:21) 
    url - /api/v1/users/admin 
    method - POST 
    IP - ::1
   {"timestamp":"2022-02-09T04:42:38.008Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T04:54:25.174Z"}
info: Connected to the database. {"timestamp":"2022-02-09T04:54:25.238Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T05:00:48.278Z"}
info: Connected to the database. {"timestamp":"2022-02-09T05:00:48.323Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T05:02:02.660Z"}
info: Connected to the database. {"timestamp":"2022-02-09T05:02:02.726Z"}
error: 
    status - 500 
    message - ReferenceError: redis is not defined
    at _callee2$ (/home/saeb/Book_Store/Book_Store/build/controllers/bookstore.controller.js:225:15)
    at tryCatch (/home/saeb/Book_Store/Book_Store/node_modules/regenerator-runtime/runtime.js:63:40)
    at Generator.invoke [as _invoke] (/home/saeb/Book_Store/Book_Store/node_modules/regenerator-runtime/runtime.js:294:22)
    at Generator.next (/home/saeb/Book_Store/Book_Store/node_modules/regenerator-runtime/runtime.js:119:21)
    at asyncGeneratorStep (/home/saeb/Book_Store/Book_Store/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/home/saeb/Book_Store/Book_Store/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/searchbook/Data%20Sc 
    method - GET 
    IP - ::1
   {"timestamp":"2022-02-09T05:06:51.676Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-02-09T05:07:16.891Z"}
info: Connected to the database. {"timestamp":"2022-02-09T05:07:16.938Z"}
